velox/exec/tests/utils/AggregationFuzzer.cpp

```diff
@@ -629,6 +635,13 @@ std::vector<RowVectorPtr> AggregationFuzzer::generateInputData(
   return input;
 }
 
+RowVectorPtr AggregationFuzzer::generateEmptyInputData(
+    std::vector<std::string> names,
+    std::vector<TypePtr> types) {
+  auto inputType = ROW(std::move(names), std::move(types));
+  return vectorFuzzer_.fuzzRow(inputType);
```

Comment:
@Krishna-Prasad-P-V could you please share why the row size of 0 was removed here? The default size used would be 100 from `opts_.vectorSize` in this case which would not help test the desired scenario. Please CMIW.

Reply:
What does the containerAvgLength mean? 
```
size_t containerAvgLength = std::max(elementsSize / size, 1UL);
```

When size is 0, it fails with division by zero error.

Reply:
Thanks for sharing @Krishna-Prasad-P-V, I hadn't encountered this error before when testing. Reverted this function now and using vectorSize in fuzzer options instead to generate empty inputs.